pkg/server/api.go

```diff
@@ -111,3 +112,21 @@ func (sh *APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		glog.Errorf("couldn't encode the config for req: %v, error: %v", cr, err)
 	}
 }
+
+type healthHandler struct{}
+
+// ServeHTTP handles /healthz requests.
+func (h *healthHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "GET" {
+		w.WriteHeader(http.StatusMethodNotAllowed)
+		return
+	}
+
+	if r.URL.Path == "" {
```

Comment:
what is this check capturing.. ?

Reply:
also the path is going to be set to  `/healthz` based on setup here https://github.com/openshift/machine-config-operator/pull/239/commits/628e654b1e9a42ce30b11e618146dd051cc3bdad#diff-4bbf1ce644e1b41f1ed61ab858def528R46

Reply:
Sorry, I'm trying to wrap my head around how muxers work :p.  It looks like they pass the full URL through, and aren't stripping off the portion handled by the muxer's switch.  Anyhow, don't bother looking at this PR until #240 has landed ;).

Reply:
i did some local testing and this sheds some light on Path reaginrding how the muxer works.
https://github.com/openshift/machine-config-operator/pull/242#issuecomment-447943204